internal/venice-common/src/main/java/com/linkedin/venice/store/rocksdb/RocksDBUtils.java

```diff
@@ -95,4 +99,24 @@ public static int extractTempRMDSSTFileNo(String fileName) {
     }
     return Integer.parseInt(fileName.substring(TEMP_RMD_SST_FILE_PREFIX.length()));
   }
+
+  /**
+   * Deletes the files associated with the specified store, version, and partition.
+   *
+   * @param storeName the name of the store
+   * @param version the version number of the store
+   * @param partition the partition ID
+   */
+  public static void deletePartitionDir(String baseDir, String storeName, int version, int partition) {
+    Path path = null;
+    try {
+      path = Paths.get(baseDir, storeName, String.valueOf(version), String.valueOf(partition));
+      if (Files.exists(path)) {
+        Files.walk(path).sorted(Comparator.reverseOrder()).map(Path::toFile).forEach(File::delete);
```

Comment:
Sry. One thing I missed to review last time, in the p2p client, it would create the directory if not exist. Does this method also clean up the directory or just files? We need to delete the partition directory as if nothing happens

Reply:
if the directory is `storeName/5/6/file2.txt` , it would delete `5/6/file2.txt`. Does this answer your question?